addons/html_editor/static/src/fields/html_field.js

```diff
@@ -62,6 +71,15 @@ export class HtmlField extends Component {
                 this.props.record.data[this.props.name]
             ),
         });
+        useEffect(() => {
+            if (this.props.dynamicPlaceholder) {
+                this.editor.shared.updateDphDefaultModel(
+                    this.props.record.data[
+                        this.props.dynamicPlaceholderModelReferenceField || "model"
+                    ]
+                );
+            }
+        });
```

Comment:
You juste need to call updateDphDefaultModel when props.record.data[xxx] change no ?

Reply:
yes, is there something better than useEffet() ?